release/release_after_kokoro.sh

```diff
@@ -89,7 +89,8 @@ ask_proceed
 ###############################################################################
 echo
 echo "#"
-echo "# Run Eclipse from the command line to generate the following files:"
+echo "# Run Eclipse from the command line to generate a p2 repository from"
+echo "# the signed artifacts:"
```

Comment:
I don't think the _the following p2 repository_ works as we're generating a new p2 repository.  So I'll add that.